Use `newMigrationPath` and `newMigrationNamespace` if `sourceNamespaces` and `sourcePaths` are not specified (#333)
yiisoft/yii-db-migration: CHANGELOG.md

- Enh #322, #330: Improve output of `migrate:up`, `migrate:down`, `migrate:redo`, `migrate:new`, `migrate:history`, and
  `migrate:create` commands: remove redundant messages, replace `>>>` with cleaner output, and move "Database
  connection" info to the top (@samdark, @vjik)

> - Enh #333: Use `newMigrationPath` and `newMigrationNamespace` if `sourceNamespaces` and `sourcePaths` are not specified (@Tigrov)

Reviewer: ```suggestion
- Enh #333: Use `newMigrationPath` and `newMigrationNamespace` as source (@Tigrov)
```
